README.md

```diff
@@ -1,103 +1,105 @@
 # mquery: Blazingly fast Yara queries for malware analysts
 
-Ever had trouble searching for particular malware samples? Our project is an analyst-friendly web GUI to look through your digital warehouse.
+Ever had trouble searching for particular malware samples? Mquery is an
```

Comment:
`particular` sounds kinda weird in reference to a group

After all, the definition: 
> used to single out an individual member of a specified group or class.

Reply:
> `particular` sounds kinda weird in reference to a group

I don't see any particular problems here.

Reply:
particular sounds particularly weird in a reference to particular group